docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,366 @@
+| Microsoft.Network/privateDnsZones/read               | Subscription |
+| Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete | Subscription |
+| Microsoft.Network/privateDnsZones/virtualNetworkLinks/read | Subscription |
+| Microsoft.Network/privateDnsZones/virtualNetworkLinks/write | Subscription |
+| Microsoft.Network/privateDnsZones/write              | Subscription |
+| Microsoft.Network/publicIPAddresses/delete           | Subscription |
+| Microsoft.Network/publicIPAddresses/join/action      | Subscription |
+| Microsoft.Network/publicIPAddresses/read             | Subscription |
+| Microsoft.Network/publicIPAddresses/write            | Subscription |
+| Microsoft.Network/routeTables/delete                 | Subscription |
+| Microsoft.Network/routeTables/read                   | Subscription |
+| Microsoft.Network/routeTables/write                  | Subscription |
+| Microsoft.Network/virtualNetworks/delete             | Subscription |
+| Microsoft.Network/virtualNetworks/join/action        | Subscription |
+| Microsoft.Network/virtualNetworks/join/action        | Subscription |
+| Microsoft.Network/virtualNetworks/joinLoadBalancer/action | Subscription |
+| Microsoft.Network/virtualNetworks/peer/action         | Subscription |
+| Microsoft.Network/virtualNetworks/read               | Subscription |
+| Microsoft.Network/virtualNetworks/subnets/delete     | Subscription |
+| Microsoft.Network/virtualNetworks/subnets/join/action| Subscription |
+| Microsoft.Network/virtualNetworks/subnets/joinLoadBalancer/action | Subscription |
+| Microsoft.Network/virtualNetworks/subnets/read       | Subscription |
+| Microsoft.Network/virtualNetworks/subnets/virtualMachines/read | Subscription |
+| Microsoft.Network/virtualNetworks/subnets/write      | Subscription |
+| Microsoft.Network/virtualNetworks/virtualMachines/read | Subscription |
+| Microsoft.Network/virtualNetworks/virtualNetworkPeerings/delete | Subscription |
+| Microsoft.Network/virtualNetworks/virtualNetworkPeerings/read | Subscription |
+| Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write | Subscription |
+| Microsoft.Network/virtualNetworks/write              | Subscription |
+| Microsoft.Resources/subscriptions/resourceGroups/read | Subscription |
+| Microsoft.Storage/storageAccounts/blobServices/containers/read | Subscription |
+| Microsoft.Storage/storageAccounts/blobServices/containers/write | Subscription |
+| Microsoft.Storage/storageAccounts/blobServices/listKeys/action | Subscription |
+| Microsoft.Storage/storageAccounts/read                | Subscription |
+| Microsoft.Storage/storageAccounts/write               | Subscription |
+
+
+
+
+## Validator
+
+To use the [Validator](https://github.com/spectrocloud-labs/validator) with the [Azure plugin](https://github.com/spectrocloud-labs/validator-plugin-azure) you need the following actions to be able to ensure the roles attached to the service principal used by Validator have the required permissions:
```

Comment:
When would be good cases for me to use the Validator? As you've described under static placement, that would be "when you have a pre-existing network resource group, VNet, subnets and security groups you want Palette to use when deploying a cluster". What is the use case here?